extensions/interactions/guppy.html

```diff
@@ -1,33 +1,60 @@
-<link rel="stylesheet" href="/third_party/static/guppy-b5055b/build/guppy-default.min.css">
+<link rel="stylesheet" href="/third_party/static/guppy-f94402/build/guppy-default.min.css">
+<link rel="stylesheet" href="/third_party/static/guppy-f94402/build/guppy-default-osk.min.css">
 <script>
   // Use the normal default browser eventlistener without passive set to false.
   // TODO(#8833): Use passive listeners when guppy supports it.
   EventTarget.prototype.addEventListener = (
     EventTarget.prototype.addEventListener._original);
 </script>
-<script type="text/javascript" src="/third_party/static/guppy-b5055b/build/guppy.min.js"></script>
+<script type="text/javascript" src="/third_party/static/guppy-f94402/build/guppy.min.js"></script>
+<script type="text/javascript" src="/third_party/static/guppy-f94402/build/guppy_osk.min.js"></script>
 
 <script type="text/javascript">
+  var osk = new GuppyOSK({'goto_tab':'arithmetic'});
+  Guppy.use_osk(osk);
   Guppy.init({
-    "path": "/third_party/static/guppy-b5055b/build"
+    'path': '/third_party/static/guppy-f94402/build'
+  });
+  Guppy.configure('buttons', ['osk', 'controls']);
+  Guppy.configure('empty_content', '\\color{grey}{\\text{\\small{Type a formula here.}}}')
+  document.addEventListener('keydown', function (event) {
```

Comment:
is this event listener related to the comment on lines 4 and 5?

Reply:
No -- AFAICT, since this is not related to touch-based events, it wouldn't matter if the event listener is active or passive.